content/building-blocks/automate-and-execute-your-work/reproducible-work/google_cloud_docker.md

```diff
@@ -0,0 +1,312 @@
+---
+title: "Import and Run a python environment on Google cloud with Docker" 
+description: "Learn how to import and run a python environment on google cloud with docker"
+keywords: "docker, images, container, environments, virtual, Python, Jupyter, Jupyter notebook, environment, google cloud, cloud"
+weight: 2
+author: Diego Sanchez Perez
+draft: false
+aliases:

```

Comment:
The first alias is used to make a short link. In this case I would make it something like: 

Aliases:

> - /run/jupyter-on-cloud

Reply:
Got it!